### PR TITLE
[WIP] New components and improvements

### PR DESCRIPTION
**Pull request by Drache93**

### CODE REVIEW

**Comment by miraage** [comment]
Can we move the icons package one level up?

**Comment by Drache93** [comment]
> Can we move the icons package one level up?

You mean outside of the components folder?

**Comment by miraage** [comment]
@Drache93 yeah, I was a bit surprised to see nested package. Now I can see you removed it

**Comment by Drache93** [comment]
@miraage Was trying to figure out `import {} from '@hiro-ui/bootsrap/icons'`; I missed the obvious way!

File: packages/bootstrap/src/components/elements/Flex/Box.ts [hunk 1]
```diff
@@ -72,7 +75,9 @@ export const Box = styled.div<BoxProps>(
   ({ bordered, theme }) =>
     bordered
       ? {
-          border: `1px solid ${theme.colors[bordered === true ? 'primary' : bordered].default}`,
+          border: `1px solid ${
+            theme.colors[bordered === true ? 'primary' : bordered].default
```
Comment (miraage) [inline packages/bootstrap/src/components/elements/Flex/Box.ts]: this looks like an error semantically

**Pull request merged**